payara-common/dependency-reduced-pom.xml

```diff
@@ -6,7 +6,7 @@
     <version>1.0.Beta3</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
-  <artifactId>payara-container-4-common</artifactId>
+  <artifactId>payara-container-common</artifactId>

```

Comment:
@jGauravGupta Is that also the official name in the maven central archive. As of now I can find both version there: <https://search.maven.org/search?q=arquillian-payara-server>

Reply:
Both artifacts are available to help in migration.